public/js/edit-catalog.js

```diff
@@ -93,22 +93,22 @@ window.HLRDESK.init.editCatalog = function() {
       var date = result.date_added.substring(0,10);
     }
 
-    $('#dateAdded').val(date);
+    $('#dateAdded').html(date);
 
-    $('#editedBy').val(result.edited_by);
+    $('#editedBy').html(result.edited_by);
```

Comment:
Looks good. Can you use `text` instead of `html` here? It will help prevent any XSS if someone manages to get some bad data in the database.